packages/ddex/entrypoint/src/App.tsx

```diff
@@ -30,6 +28,8 @@ const links = {
   audius: 'https://audius.co'
 }
 
+const env = import.meta.env.VITE_ENVIRONMENT as 'dev' | 'stage' | 'prod'
```

Comment:
i think the webapp dockerfile starts the app with `npm run start` so this envvar wont be present - the env can or might already be pulled from audius-docker-compose, can you use that instead?

Reply:
this is the separate distro.audius.co entry point :-)